source/MaterialXView/Viewer.cpp

```diff
@@ -37,6 +38,10 @@ const int ALBEDO_TABLE_SIZE = 64;
 const int IRRADIANCE_MAP_WIDTH = 256;
 const int IRRADIANCE_MAP_HEIGHT = 128;
 
+const int MIN_TEXTURE_RES = 256;
+const int MAX_TEXTURE_RES = 4096;
```

Comment:
For later: To be more flexible the maximum here is for OpenGL. It could be better to get this from the baker class itself.
Then for instance a software baker can set a different max resolution.